cf-agent/verify_files.c

```diff
@@ -261,6 +262,7 @@ void ClearExpandedAttributes(Attributes *a)
     free(a->transaction.measure_id);
     a->transaction.measure_id = NULL;
     free(a->edit_template);
+    free(a->edit_template_string);
```

Comment:
Set it to NULL also. No big deal, it's just in order to follow the lines around it.